Documentation/csi_relatedpubs.md

```diff
@@ -0,0 +1,27 @@
+# Related information and publications
```

Comment:
how is this file different from `csi_relatedpubs.md`?

Reply:
it's shown in different parts of the PDFs and books / online seprately.